security/data-redaction/demo.sql

```diff
@@ -0,0 +1,172 @@
+--------------------------------------------------------------------------------
+-- DEMO SCRIPT: Exploring Data Redaction Enhancements in Oracle Database 23ai
+-- Requirements:
+--   (1) Oracle Database 23ai 23.6+
+--   (2) HR sample schema installed from https://github.com/oracle-samples/db-sample-schemas/releases
+--   (3) Privileges to run DBMS_REDACT and alter HR objects (or run as HR)
+--------------------------------------------------------------------------------
+
+--------------------------------------------------------------------------------
+-- 0. Verify setup
+--------------------------------------------------------------------------------
+-- Confirm version
+SELECT * FROM v$version;
+
+-- Confirm HR schema exists
+SELECT username FROM all_users WHERE username = 'HR';
```

Comment:
If `HR` doesn't exist, some of the following statements will fail.

Ideally, the script should exit if `HR` does not exist.

Also, the requirements should also be documented in the ReadMe instead of just the script header.